loadgen/scenario.go

```diff
@@ -164,15 +183,22 @@ type KitchenSinkWorkflowOptions struct {
 	StartOptions client.StartWorkflowOptions
 }
 
-// ExecuteKitchenSinkWorkflow starts the generic "kitchen sink" workflow and waits for its completion ignoring its result.
+// ExecuteKitchenSinkWorkflow starts the generic "kitchen sink" workflow and waits for its
+// completion ignoring its result.
 func (r *Run) ExecuteKitchenSinkWorkflow(ctx context.Context, options *KitchenSinkWorkflowOptions) error {
-	r.Logger.Debugf("Executing workflow with options: %v", options.StartOptions)
-	execution, err := r.Client.ExecuteWorkflow(ctx, options.StartOptions, "kitchenSink", options.Params)
+	return r.ExecuteAnyWorkflow(ctx, options.StartOptions, "kitchenSink", nil, options.Params)
+}
+
+// ExecuteAnyWorkflow wraps calls to the client executing workflows to include some logging,
+// returning an error if the execution fails.
+func (r *Run) ExecuteAnyWorkflow(ctx context.Context, options client.StartWorkflowOptions, workflow interface{}, valuePtr interface{}, args ...interface{}) error {
+	r.Logger.Debugf("Executing workflow %s with options: %v", workflow, options)
```

Comment:
nit: This is noisy since it logs for every workflow spawned, I'd rather keep debug level a bit cleaner than this.

Reply:
That was already the case?